Window groups (tabs) support in Hyprland

pilgren opened the pull request:
### Add Hyprland group support with theme styling

- Added group window (tabs) support with SUPER + SHIFT + T keybinding
- Added keybinding for SUPER + Tab to cycle through grouped windows
- Implemented theme-aware group bar colors that match each theme

rafaelbernard commented:
Nice. See also https://github.com/basecamp/omarchy/pull/656

rafaelbernard commented:
For `SUPER + TAB`, something else is also claiming for that. :-P
https://github.com/basecamp/omarchy/pull/652

pilgren commented:
@rafaelbernard Ahh yes, I didn't see that other PR. I guess that the other PR has a bit more configurations than this one :smiley:

pilgren commented:
@dhh Yes, of course.

Here are the tabs in a couple of the different themes:

Tokyo Night:
<img width="3840" height="2160" alt="image" src="https://github.com/user-attachments/assets/f16b01dd-9627-4cd7-b362-38195ed5fd26" />

Catppuccin Latte:
<img width="3840" height="2160" alt="image" src="https://github.com/user-attachments/assets/f35c612d-af26-4613-8abd-c3733d1b1055" />

Everforest:
<img width="3840" height="2160" alt="image" src="https://github.com/user-attachments/assets/48170b18-5196-4069-8d21-9509c7298935" />

bastnic commented:
If you're interested in tabs, you might consider a sway-like approach: toggle all active windows into a single tab group,  perfect when switching between an external monitor and a laptop on the go.

👉 Demo video: https://github.com/basecamp/omarchy/discussions/326

Benefit: it reuses the existing SUPER + ← / → bindings to navigate between grouped windows.

bastnic commented on default/hypr/bindings/tiling.conf:
```diff
@@ -7,6 +7,10 @@ bindd = SUPER, P, Pseudo window, pseudo, # dwindle
 
+# Group windows
+bind = SUPER SHIFT, T, Toggle tabs, togglegroup
+bind = SUPER, Tab, Cycle to next window in tabs, changegroupactive
```
are you sure it's not `bindd`? 

using `bind` will get `invalid dispatcher error`

dkeysil commented:
In Sway it was amazing that inside tabbed group you could split window vertically or horizontally, so you would have 2 or more apps simultaneously

dkeysil commented:
Also would be cool to have contextual SUPER + arrow, because in Sway you can use the same SUPER + Right/Left to navigate inside tabbed group and just inside workspace

Here I see that we need to bind different key binding for navigating inside tabbed group

pilgren commented:
I have updated the key bindings to avoid `SUPER + TAB` which is now used elsewhere.

I also added some of the same bindings as @mfornasa mentioned and support to navigate inside tabbed group with `SUPER ALT + Right/Left` as suggested by @dkeysil

Here is a small recording of how it looks:

https://github.com/user-attachments/assets/8402781e-c252-4846-8f87-d065c1b53da0

dhh commented:
Solved via #656
